lessons/pydata/api/index.ipynb

```diff
@@ -1374,7 +1374,9 @@
   {
```

Comment:
<p>Requests by už měli mít nainstalováno (v úvodní lekci + se to používalo pro stahování dat v EDA), navíc tuhle knihovnu importuješ už o několik buněk výš. Ale jinak bych doporučil spíš takto:</p><p> </p><p> </p><p><code>%pip install requests </code></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/PyDataCZ/naucse.python.cz/pull/122/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='97'/>

Reply:
Opraveno.